Tests/MongoSwiftSyncTests/SpecTestRunner/TestOperation.swift

```diff
@@ -1,1109 +0,0 @@
-import MongoSwiftSync
```

Comment:
This file was getting unwieldy, so I split it up into a bunch of different files and moved them to the `TestOperations/` subdirectory.